### PR TITLE
feat: added result page

### PR DESCRIPTION
**Pull request by Shubhamkr2610**
I have created result page ,  added some responsive css for  mobile view responsive ,  also refactored some code in database that you can ignore  
files to check 
src/pages/result-page/Result.jsx
src/pages/result-page/Result.css
src/pages/quiz-page/Quiz.jsx
src/pages/quiz-page/quiz.css

image

### CODE REVIEW

File: src/context/quizContext.js [hunk 1]
```diff
@@ -8,7 +8,9 @@ const quizContext = createContext()
 
 
  const Quizprovider = ({children}) =>{
-     const [ questionCollection ,  setQuestionCollection ] = useState([])
+     const [ questionCollection ,  setQuestionCollection ] = useState([]);
+     const [score, setScore] = useState(0);
+     const [result , setResult ] = useState([]);
```
Comment (sanjitsarkar) [inline src/context/quizContext.js:13]: You can  create a function for getting score on clicking correct option for an question.
Name the Context with PascalCase

File: src/pages/quiz-page/Quiz.jsx [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 import './quiz.css';
 export const Quiz = () => {
-  const { questionCollection } = useQuiz()
+
+  const navigate = useNavigate();
+  const { questionCollection, score, setScore , setResult , result} = useQuiz()
+
+
   const [questionindex, setQuestionIndex] = useState(0);
   const [question, setQuestion] = useState("");
```
Comment (sanjitsarkar) [inline src/pages/quiz-page/Quiz.jsx:12]: Extract these states to provider

**Review by sanjitsarkar — APPROVED** [review]
Everything looks good and Good to merge

File: src/context/quizContext.js (hunk 1 above)
Comment (Shubhamkr2610) [reply src/context/quizContext.js:13]: okay , I will do it for the time being , As I have to submit and I am running out of time

File: src/pages/quiz-page/Quiz.jsx (hunk 2 above)
Comment (Shubhamkr2610) [reply src/pages/quiz-page/Quiz.jsx:12]: good pick , will do it

**Review by arjun-computer-geek — APPROVED** [review]
Code looks good. Good code of conduct.

**Review by vishalg8454 — APPROVED** [review]
All looks good, can merge.

**Pull request merged**